pages/02.Setup/01.Getting Started/docs.en.md

```diff
@@ -109,23 +109,23 @@ The next step after creating the database is to set up your administrator accoun
 
 ![admin user](admin-user.png)
 
-Enter a username, password, first and last name and an email address, and the account will be created for you.
+Enter a username, password, first and last name, and an email address, and the account will be created for you.
```

Comment:
See above re. serial comma, however this is a super long sentence when you read it out loud, do you think we could write this differently? 

Do we need to list out the fields individually or could we refer to them collectively (for example, 'enter the required fields and the account will be created for you')?  What do you think @fakela ?

Reply:
_Enter the required fields: username, password, first and last name, email address, and the account would be created for you_

@RCheesley I referred to them collectively do you think this is okay?

Reply:
Yes, perfect!

Reply:
Also, I have done this